File: Day249/findTheDifference.java
package Day249;

public class findTheDifference {
    public static char diff(String s, String t) {
        int temp = 0;
        for (int i=0; i<s.length(); i++) {
            temp+=(int)s.charAt(i);
        }
        for (int i=0; i<t.length(); i++) {
            temp -= (int)t.charAt(i);
        }
        return (char)Math.abs(temp);
    }
    public static void main(String[] args) {
        String s = "abcd";
        String t = "abcde";
        System.out.println(diff(s, t));
    }
}

File: Day249/findTheDifference.go
package main 

import (
	"fmt"
	"math"
)
func findTheDifference(s string, t string) byte {
    sSum, tSum := 0, 0
    for i:=0; i<len(s); i++ {
        sSum += int(s[i])
    }
    for i:=0; i<len(t); i++ {
        tSum += int(t[i])
    }
    // fmt.Println(sSum, tSum)
    return byte(math.Abs(float64(sSum)-float64(tSum)))
}


func main() {
	s := "abcd"
	t := "abcde"
	fmt.Println(string(findTheDifference(s, t)))
}